create_live_iso

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+bas64_str=`base64 -w 0 handle-user-data`
+
+sed -i.back "/^cat > handle-user-data.base64 << EOF/{s/.*/&\n$bas64_str/;:a;n;ba}" centos-7-minimal.ks
```

Comment:
I'd rater rename the current kickstart file to something like _centos-7-minimal.ks.template_ which then get process to create _centos-7-minimal.ks _. Inline replacing will modify original file.

Will this even work properly when called multiple times in a row?